include/maliput/api/lane.h

```diff
@@ -237,6 +240,8 @@ class Lane {
   virtual const LaneEndSet* DoGetOngoingBranches(const LaneEnd::Which which_end) const = 0;
 
   virtual std::optional<LaneEnd> DoGetDefaultBranch(const LaneEnd::Which which_end) const = 0;
+
+  virtual LaneType DoType() const = 0;
```

Comment:
out of curiosity, what are these `Do...` methods (above)? Also, why are some snake case and some camel case?